docs/word/citation-management.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Sample: Manage citations through your Word add-in'
+description: Learn how to create an add-in that manages citations in a Word document.
+ms.date: 11/15/2023
+ms.topic: sample
+ms.localizationpriority: medium
+---
+
+# Sample: Manage citations in a Word document using your Word add-in
+
+Citation management is an important aspect of documents, particularly in academia and education. Each citation style has its own guidelines for how citations should be marked in a document as well as where and how the sources should be noted. Examples of citation styles are [APA](http://apastyle.apa.org/) and [MLA](https://www.mla.org/MLA-Style).
+
+This article shows how to create an add-in that showcases the basics of managing citations in a Word document. The add-in displays the references loaded from the user's .bib file then the user can insert selected references into the document.
+
+![Screen showing citation management add-in task pane.](../images/word-citation-management.png)
+
+## Prerequisites
+
+- [Visual Studio Code](https://code.visualstudio.com/Download).
+- Office connected to a Microsoft 365 subscription (including Office on the web).
+- [Node.js](https://nodejs.org/) version 16 or greater.
+- npm version 8 or greater.
+- @orcid/bibtexParseJs, a JavaScript library for parsing the selected .bib file.
+
+## Run the sample code
+
+The sample code for this article is named [Manage citations in a Word document](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/word-citation-management). To run the sample, follow the instructions in the [readme](https://github.com/OfficeDev/Office-Add-in-samples/tree/main/Samples/word-citation-management).
+
+## Key steps in the sample
+
+1. The user chooses a local .bib file that contains the references they'd like to cite.
+1. The add-in reads the .bib file then displays the bibliography references in the task pane. The sample uses [@orcid/bibtexParseJs](https://github.com/ORCID/bibtexParseJs#readme) to parse the .bib file.
+1. The user chooses the appropriate reference then inserts it at the cursor's location (or at the end of selected text) in the document.
+1. The add-in adds a reference mark at that location in the document and adds the reference to an endnote. All endnotes are automatically listed at the end of the document.
+
+## Additional considerations for your solution
```

Comment:
I liked the "Make it your own" tone from the other readme.  Can we bring that vibe here?